application/classes/Ushahidi/Repository/Post/Video.php

```diff
@@ -0,0 +1,23 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi Post Video Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2016 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Entity\PostValue;
+use Ushahidi\Core\Entity\PostValueRepository;
+
+class Ushahidi_Repository_Post_Video extends Ushahidi_Repository_Post_Value
```

Comment:
Alternately rather than mapping to post_text, you could just extend Repository_Post_Text here...